pkg/kando/location_delete.go

```diff
@@ -32,43 +29,20 @@ func newLocationDeleteCommand() *cobra.Command {
 		Short: "Delete artifacts from s3-compliant object storage",
 		// TODO: Example invocations
 		RunE: func(c *cobra.Command, args []string) error {
-			return runLocationDelete(c)
+			if err := validateCommandArgs(c); err != nil {
+				return err
+			}
+			dataMover, err := dataMoverFromCMD(c, kopiaSnapshotFlagName)
+			if err != nil {
+				return err
+			}
+			return dataMover.Delete(context.Background(), pathFlag(c))
```

Comment:
Can we get the context from `cmd.Context()`? https://pkg.go.dev/github.com/spf13/cobra#Command.Context
[nit]

Reply:
@r4rajat from the docs, it states that `Execute()` sets the context to context.Background() if not set explicitly. - https://pkg.go.dev/github.com/spf13/cobra#Command.Context
Can you please check and confirm?

Reply:
Implemented !

Reply:
Yeah it's working !